stubs/dateparser/dateparser/languages/loader.pyi

```diff
@@ -0,0 +1,28 @@
+from collections import OrderedDict
+from typing import Any, Iterator, List, Optional, Union
+
+from ..data import language_locale_dict as language_locale_dict, language_order as language_order
+from .locale import Locale as Locale
+
+LOCALE_SPLIT_PATTERN: Any
+
+class LocaleDataLoader:
+    def get_locale_map(
+        self,
+        languages: Optional[List[str]] = ...,
+        locales: Optional[List[str]] = ...,
+        region: Optional[str] = ...,
```

Comment:
Nit: Please use the newer `str | None` syntax instead of `Optional[str]`. (The stubgen in newer mypy versions uses it too.)